### PR TITLE
Set `max-height` for story-editor container

### LINKED ISSUE #10599: Unbounded Editor
## Bug Description

If Story Editor is not bound by a div with a well-defined height then it keeps on stretching while loading more media.

https://user-images.githubusercontent.com/53055971/154408072-1e7a314e-ef82-4312-912b-cee77b5abbf6.mp4

## Steps to Reproduce

Use story editor as below
```jsx
const App = () => {
  const config = {
    apiCallbacks: { saveStoryById: () => Promise.resolve({}) },
  };

  return (
    <StoryEditor config={config} initialEdits={{ story: {} }}>
      <InterfaceSkeleton />
    </StoryEditor>
  );
};
```

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

### PR DESCRIPTION
**Pull request by mohdsayed**
## Context

Set `max-height` for story editor. See https://github.com/GoogleForCreators/web-stories-wp/issues/10599

## Testing Instructions

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

## Checklist

- [ ] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [ ] I have verified accessibility to the best of my abilities ([docs](https://github.com/googleforcreators/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [ ] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/googleforcreators/web-stories-wp/tree/main/docs#testing))
- [ ] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes https://github.com/GoogleForCreators/web-stories-wp/issues/10599

### CODE REVIEW

File: .storybook/stories/playground/dashboard/index.js [hunk 1]
```diff
@@ -31,10 +30,6 @@ export default {
   title: 'Playground/Dashboard',
 };
 
-const AppContainer = styled.div`
-  height: 100vh;
```
Comment (swissspidy) [inline .storybook/stories/playground/dashboard/index.js:35]: To clarify, we don't need a `max-height: 100vh` for the dashboard?

File: .storybook/stories/playground/dashboard/index.js [hunk 2]
```diff
@@ -108,11 +103,9 @@ export const _default = () => {
   };
 
   return (
-    <AppContainer>
-      <Dashboard config={config}>
-        <GlobalStyle />
-        <InterfaceSkeleton />
-      </Dashboard>
-    </AppContainer>
+    <Dashboard config={config}>
```
Comment (spacedmonkey) [inline .storybook/stories/playground/dashboard/index.js:106]: Don't we need to apply styling of dashboard?

File: .storybook/stories/playground/dashboard/index.js (hunk 1 above)
Comment (mohdsayed) [reply .storybook/stories/playground/dashboard/index.js:35]: I tried, by adding a lot of stories in the dashboard without `height: 100vh` in storybook, it didn't have any issue.

File: .storybook/stories/playground/dashboard/index.js (hunk 2 above)
Comment (mohdsayed) [reply .storybook/stories/playground/dashboard/index.js:106]: https://github.com/GoogleForCreators/web-stories-wp/pull/10601#discussion_r809150060

**Review by swissspidy — APPROVED** [review]

**Pull request merged**